cubeb-core/src/lib.rs

```diff
@@ -289,6 +289,7 @@ impl Binding for StreamParams {
 }
 
 /// Audio device description
+#[derive(Copy, Clone, Debug)]
```

Comment:
How is `Copy` safe when this contains a raw pointer? It seems like it'd be possible to call Stream::destroy_device on multiple copies accidentally, e.g. https://play.rust-lang.org/?gist=5ecf2f4aaf54f6f77eef5eb33238302d&version=stable